src/main/java/com/example/database/services/StoreService.java

```diff
@@ -71,4 +98,12 @@ public ItemModel getItemModel(Long id) {
                 .build();
     }
 
+    private boolean categoryCheck(Set<Category> categories) {
```

Comment:
It would be better to name this method with more business descriptive name, ex. `validUserProductCategory`
It may also be more readable what its happening there:
```suggestion
private boolean validUserProductCategory(Set<Category> categories) {
    return Stream.of(categories).noneMatch(c -> "Shipping".equalsIgnoreCase(c.getCategoryName());
}
```

Reply:
Changed, much better!